packages/salesforcedx-vscode-core/src/commands/util/toolingDeployResultParser.ts

```diff
@@ -0,0 +1,129 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import { CompositeCliCommandExecution } from '@salesforce/salesforcedx-utils-vscode/out/src/cli';
+import {
+  Row,
+  Table
+} from '@salesforce/salesforcedx-utils-vscode/out/src/output';
+import { channelService } from '../../channels';
+import {
+  DeployResult,
+  DeployStatusEnum,
+  ToolingRetrieveResult
+} from '../../deploys';
+import { nls } from '../../messages';
+
+export class ToolingDeployParser {
+  public result: ToolingRetrieveResult;
+
+  constructor(deployResult: ToolingRetrieveResult) {
+    this.result = deployResult;
+  }
+
+  public buildSuccesses(componentSuccess: DeployResult) {
+    const success = [
+      {
+        state: 'Add',
+        fullName: componentSuccess.fullName,
+        type: componentSuccess.componentType,
+        filePath: componentSuccess.fileName
```

Comment:
`componentSuccess.fileName` doesn't include the package directory part of the path, so you'll have `classes/test.cls` instead of the regular `my-package/main/default/classes/test.cls`

Reply:
Yeah, I also haven't tested this fully on an org & project with properly configured namespaces. I'll look into that next.